f2/src/summary/InformationSummary.js

```diff
@@ -62,20 +112,33 @@ export const InformationSummary = props => {
           <Stack as="dl" spacing={4} shouldWrapChildren>
             <DescriptionListItem
               descriptionTitle="confirmationPage.personalInformation.typeOfInfoReq"
-              description={intl.formatList(
-                personalInformation.typeOfInfoReq.map(i => i18n._(i)),
-                { type: 'conjunction' },
-              )}
+              description={infoReqLine}
             />
 
+          ) : (
+              <Text>
+                <Trans id="confirmationPage.impactIntro" />
+              </Text>
+            )}
+
+
             <DescriptionListItem
               descriptionTitle="confirmationPage.personalInformation.typeOfInfoObtained"
-              description={intl.formatList(
-                personalInformation.typeOfInfoObtained.map(i => i18n._(i)),
-                { type: 'conjunction' },
-              )}
+              description={infoObtainedLine}
             />
 
+          ) : (
+              <Text>
+                <Trans id="confirmationPage.impactIntro" />
+              </Text>
+            )}
+          <DescriptionListItem
+            descriptionTitle="confirmationPage.personalInformation.tellUsMore"
+            description={personalInformation.tellUsMore}
+          />
+        </Stack>
+
```

Comment:
These lines are duplicated below and are causing your syntax error. (probably when you were fixing a merge conflict 😛 )

Reply:
fixed